### DISCUSSION TITLE
Please tell me your hometown and your favorite food there.

### DISCUSSION THREAD

**Discussion by yamamOtOaina** [NAIST-SE/AttractingContributors#1119] [2024 Attraction Contributions]
Please tell me your hometown and your favorite foods there!

Link >>  https://github.com/yamamOtOaina/AttractingContributors/blob/main/README.md

**Comment by mustafizur-r** [comment]
Hello [@yamamOtOaina](https://github.com/yamamOtOaina) , I recently made a contribution to your project , I have added “Bangladesh, Biriyani food" could you please contribute to my as well ? Following is the link to [my repository](https://github.com/MustafizurRU/AttractingContributors-My_Favorite_Sci-Fi_Books) . Thanks in advance.

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140

**Comment by polarbearpppp** [comment]
Hi, I'm contribute to your project , Would you mind to contribute my project 
https://github.com/polarbearpppp/AttractingContributors-Games

**Comment by thapakorn604** [comment]
Hi, I create the pull request to your project -> https://github.com/yamamOtOaina/AttractingContributors/pull/7
if you don't mind please participate to mine -> https://github.com/NAIST-SE/AttractingContributors/discussions/1115

**Comment by Tz-violet** [comment]
hi, I just joined your project.
can you help me to complete my project: https://github.com/Tz-violet/AttractingContributors.git ?

**Comment by kozmr** [comment]
Hi, I contributed to your project.
Could you contribute to my project?

https://github.com/kozmr/AttractingContributors-YouTube
Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1113
